layout/rowwrap.go

```diff
@@ -0,0 +1,88 @@
+package layout
+
+import (
+	"fyne.io/fyne/v2"
+	"fyne.io/fyne/v2/theme"
+)
+
+type rowWrapLayout struct {
+	rowCount          int
+	horizontalPadding float32
+	verticalPadding   float32
+}
+
+// NewRowWrapLayout returns a layout that dynamically arranges objects
+// with the same height in rows and wraps them as necessary.
+//
+// Object visibility is supported.
```

Comment:
"Object visibility is supported" probably isn't needed, since all first-party layouts support object visibility, and as such it can just be assumed.